src/config.ts

```diff
@@ -73,14 +102,19 @@ export const rollupChain = defineChain({
     ...chainConfig.contracts,
     l2OutputOracle: {
       [parentChain.id]: {
-        address: import.meta.env.VITE_L2_OUTPUT_ORACLE_ADDRESS,
+        address: '0x74Ad6E0FB793eB5e6c1ff1225B03F5C5fFB7EF0c',
       },
     },
     portal: {
       [parentChain.id]: {
-        address: import.meta.env.VITE_L1_OPTIMISM_PORTAL_ADDRESS,
+        address: '0x0d605bb7d4FB586eAB750205F5247825F4D8AF4B',
       },
     },
+    // multicall3: {
+    //   [parentChain.id]: {
+    //     address: '0xa1a858ad9041B4741e620355a3F96B3c78e70ecE',
+    //   },
+    // },
```

Comment:
I'd be open to removing this contract mapping entirely if it's causing some sort of issue. Please elaborate.